### DISCUSSION TITLE
Military conflict and the impact on RavenDB operational capabilities?

### DISCUSSION THREAD

**Discussion by mcrio** [ravendb/ravendb#19012] [General]
I was pitching RavenDB for a project and a question came up about the ongoing military conflict in Israel and its impact on the operational capabilities of RavenDB HQ, and its services like RavenDB cloud. Conflict is never good, and I hope this ends asap. Is there any official statement about these concerns?

**Comment by ppekrol** [comment]
Hi,

For years, our development and operations teams have been distributed between Israel and Poland. Both locations boast experienced developers and operations experts who can step in during conflict, with high-level management also spread across these regions. We have a Business Continuity Plan (BCP) in place for such situations, ensuring that this aspect is not overlooked. 

However, we sincerely hope that we will never need to implement this procedure, but for almost a year now due to the ongoing military conflict in Israel we were testing for short periods of time the established procedures without any visible impact to our customers.
